spec/javascripts/modules/remove-filter-spec.js

```diff
@@ -54,10 +54,24 @@ describe('remove-filter', function () {
   )
 
   var $facets =
-    '<select id="level_one_taxon" name="level_one_taxon">' +
-      '<option value="">All topics</option>' +
-      '<option value="ba3a9702-da22-487f-86c1-8334a730e559">Entering and staying in the UK</option>' +
-    '</select>' +
+    '<div id="level_one_taxon">' +
+      '<select name="level_one_taxon">' +
+        '<option value="">All topics</option>' +
+        '<option value="aa3a9702-da22-487f-86c1-8334a730e558">Entering and staying in the UK</option>' +
+      '</select>' +
+    '</div>' +
+    '<div id="level_two_taxon">' +
+      '<select name="level_two_taxon">' +
+        '<option value="">All topics</option>' +
+        '<option value="bb3a9702-da22-487f-86c1-8334a730e559">Entering and staying in the UK</option>' +
+      '</select>' +
+    '</div>' +
+    '<div>' +
+      '<select name="dropdown" id="dropdown">' +
```

Comment:
This looks like a duplication of the 'All topics' select element. In reality I think there'd only ever be two selects, but there are three here now - why?

Reply:
it's to keep the tests happy, basically the same reason as the above comment. They don't need to be `selects` but there needs to be some element with the correct `name` and `id` for the tests not to fail

Reply:
Fixed after a chat :)